perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -83,6 +83,7 @@ public static boolean authorizedInternal(PerunSession sess, String policyDefinit
 	 * @return true if the principal authorized, false otherwise
 	 * @throws InternalErrorException if something goes wrong
 	 */
+	@Deprecated
```

Comment:
What about ``isAuthorized(PerunSession sess, String role)`` at line 350/351. Shouldn't it be deprecated too?

Reply:
Fixed